dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp

```diff
@@ -593,6 +595,8 @@ createBusBreakerNetwork(const BusBreakerNetworkProperty& properties) {
           .setRho(1.)
           .endStep()
           .add();
+      transformer.getLeg1().getRatioTapChanger().setRegulationTerminal(stdcxx::ref(transformer.getLeg1().getTerminal()));
```

Comment:
Does not change the tests?

Reply:
I will have another look at it, maybe it is not needed anymore